pdp/dao.py

```diff
@@ -66,27 +94,35 @@ def get_name(netid):
     return name
 
 
-def get_student(netid):  # TODO fix this to use student id
+# uwhr is default--we can override to use HEPPS for now
+def get_employee(eid, source='uwhr'):
     """
-    Look up IRWS student data by netid (for mock...system key in real life).
+    Look up IRWS employee data by eid
+    defaults to uwhr--override to hepps until HRP finishes
     """
     try:
-        # TODO fix this to use student id
-        sdb_person = IRWS().get_sdb_person(sid=netid)
-        if not sdb_person:
-            raise ServiceError('no SDB entry')
+        # person = IRWS().get_person() # don't need?
+        uwhr_person = IRWS().get_uwhr_person(eid=eid)
+        if source == 'hepps':
+            uwhr_person = IRWS().get_hepps_person(eid=eid)
+        if not uwhr_person:
+            raise ServiceError('no ' + source + ' entry')
     except Exception as e:
         raise ServiceError(e)
-    return sdb_person
+    return uwhr_person
 
 
-def get_student(netid):  # TODO fix this to use student id
+def get_student(system_key):  # TODO fix this to use student id
     """
     Look up IRWS student data by netid (for mock...system key in real life).
     """
     try:
-        # TODO fix this to use student id
-        sdb_person = IRWS().get_sdb_person(sid=netid)
+        # Strictly speaking, iam-restools seems to want a studentid (sid)
+        # but system_key is the "official" IRWS validid for a student.
+        # IRWS will look up a student in the SDB source with either one,
+        # meaning the URLS are the same, so I am feeding the method
+        # a system key (mattjm 2016-05-06)
+        sdb_person = IRWS().get_sdb_person(sid=system_key)
```

Comment:
For this and get_uwhr_person I'd like an optional uri=None argument that takes an identifier uri and loads that. It would get us out of having to pick out 9 digits (always 9? what if the something changes in irws?) and track which source we want. Not a change I expect you to have to make, however.